subjects/cybersecurity/image-inspector/README.md

```diff
@@ -0,0 +1,134 @@
+## Image-Inspector
+
+<center>
+<img src="./resources/steganography-meme.png?raw=true" style="width: 600px !important; height: 516px !important;"/>
+</center>
+
+### Introduction
+
+Images can contain more than just visual information, they often carry hidden data such as metadata or even concealed messages through techniques like steganography. This project is designed to help you explore and analyze these hidden aspects of images.
+
+### Objective
+
+The goal is to develop a tool using a programming language of your choice (Python is recommended) that can analyze images to extract hidden information. Specifically, your tool should:
+
+1. **Extract Metadata**: Identify and display metadata from images, such as geolocation (latitude and longitude) where the photo was taken, the device used, and other relevant information.
+2. **Detect Steganography**: Discover and extract any hidden PGP keys or other data concealed within the image using steganography techniques.
+
+By completing this project, you will:
+
+- Develop an understanding of image recognition techniques.
+- Gain practical experience with steganography and metadata extraction.
+- Learn to identify hidden information in images, which can be crucial in cybersecurity investigations.
+
+### Resources
+
+Some useful resources:
+
+- [Steganography](https://en.wikipedia.org/wiki/Steganography)
+- [Exif Metadata](https://en.wikipedia.org/wiki/Exif)
+- [Python Imaging Library (PIL)](https://pillow.readthedocs.io/en/stable/)
+
+Before asking for help, explore these resources to fully understand the concepts behind image analysis and steganography.
+
+### Role Play
+
+To enhance your learning experience and assess your knowledge, a role play question session will be included as part of this project. This session will involve answering a series of questions in a simulated real-world scenario where you assume the role of a digital forensics expert explaining how your tool identifies hidden information in images.
+
+The goal of the role play question session is to:
+
+- Assess your understanding of image analysis and steganography.
+- Test your ability to communicate effectively and explain the techniques used in your project.
+- Challenge you to think critically about the implications of hidden data in images.
+
+Prepare for a role play question session in the audit.
+
+### Project Requirements
+
+#### Metadata Extraction:
+
+Your tool should be able to extract and display key metadata from an image file, including but not limited to:
+
+- Geolocation (Latitude and Longitude)
+- Device information (camera make/model)
+- Date and time the photo was taken
+
+#### Steganography Detection:
+
+- Detect hidden PGP keys or other concealed information in the image using steganography techniques.
```

Comment:
The hidden information in this example is stored inside the file's content, making it hard to detect if it's using different techniques.
so that's why we are aiming on PGP only.

Reply:
Yes I'm not saying the opposite i just asked you if you can specify `PGP` in that line instead of saying `steganography techniques`

Reply:
the used technique is steganography techniques but the hidden content is a PGP, so there is no need to change it.

Reply:
Done in this PR: https://github.com/01-edu/public/pull/2731